data_juicer/utils/model_utils.py

```diff
@@ -553,6 +553,12 @@ def prepare_recognizeAnything_model(
     return model
 
 
+def prepare_fastsam_model(pretrained_model_name_or_path):
+    from ultralytics import FastSAM
+
+    return FastSAM(pretrained_model_name_or_path)
```

Comment:
how about downloading the checkpoint automatically for users? If not, we'd better add a comment in config_all.yaml and doc string of this OP to tell users how to download them manually.

Reply:
I set the default model path to 'FastSAM-x.pt,' and when the user does not provide a model path, **the ultralytics library will automatically downloads the model**. When the model weight are not found in the specified path, the ultralytics library will also automatically download it.